src/Common/src/System/Security/Cryptography/ECDiffieHellmanOpenSslPublicKey.cs

```diff
@@ -91,6 +99,26 @@ internal SafeEvpPKeyHandle DuplicateKeyHandle()
                     throw;
                 }
             }
+
+            private void ThrowIfDisposed()
```

Comment:
would it make sense (or is it feasible) to put this method to a base class?

Reply:
It'd have to be on a base class in a different assembly, so it'd need to be declared protected (or public).  That's a bigger/different change :smile:.